app/routes/api/v1/warehouse/customers/$customerId/index.ts

```diff
@@ -0,0 +1,23 @@
+import { Customer } from "@prisma/client";
+import { LoaderFunction } from "@remix-run/node";
+import { badRequest, notFoundRequest } from "~/helpers/response-helpers.server";
+import { findCustomer } from "~/models/customer/customer.server";
+
+type LoaderData = Customer;
+export const loader: LoaderFunction = async ({
+  params,
+}): Promise<LoaderData | null> => {
+  const customerId = params.customerId;
+
+  if (customerId == null) {
+    throw badRequestResponse();
+  }
+
+  const customer = await findCustomer(customerId);
+
+  if (customer == null) {
+    throw notFoundResponse();
+  }
+
+  return customer;
+};
```

Comment:
All of this is not necessary.

The `id` will be retrieved from the jwt once logged in.

Please refactor this route